apps/core-v2-testnet/src/utils/validateAddresses.ts

```diff
@@ -0,0 +1,11 @@
+import { ethers } from "hardhat";
+import { throwInvalidAddressError } from "src/errors";
+
+export function validateAddresses(addresses: string[]): void | never {
```

Comment:
this is neat, add a TODO to move it into @elementfi/base?

Reply:
nice suggestion